twitchio/client.py

```diff
@@ -120,6 +125,7 @@ def from_client_credentials(
         self = cls.__new__(cls)
         self.loop = loop or asyncio.get_event_loop()
         self._http = TwitchHTTP(self, client_id=client_id, client_secret=client_secret)
+        self._heartbeat = heartbeat
```

Comment:
Ideally this would best be done in its own pull request, as it's not relevant to this one